src/runtime/virtcontainers/sandbox.go

```diff
@@ -746,51 +753,30 @@ func (s *Sandbox) coldOrHotPlugVFIO(sandboxConfig *SandboxConfig) (bool, error)
 			}
 
 			for _, dev := range cdiSpec.Linux.Devices {
-				isVFIODevice := deviceManager.IsVFIODevice(dev.Path)
-				if hotPlugVFIO && isVFIODevice {
-					vfioDev := config.DeviceInfo{
-						ColdPlug:      true,
-						ContainerPath: dev.Path,
-						Port:          sandboxConfig.HypervisorConfig.HotPlugVFIO,
-						DevType:       dev.Type,
-						Major:         dev.Major,
-						Minor:         dev.Minor,
-					}
-					if dev.FileMode != nil {
-						vfioDev.FileMode = *dev.FileMode
-					}
-					if dev.UID != nil {
-						vfioDev.UID = *dev.UID
-					}
-					if dev.GID != nil {
-						vfioDev.GID = *dev.GID
-					}
-
-					vfioDevices = append(vfioDevices, vfioDev)
+				if !deviceManager.IsVFIODevice(dev.Path) {
```

Comment:
When the PCIePort is `NoPort`, following this code change, it seems that origin logics are changed. Or do I miss something import ?

Reply:
I think you're right. We'll probably also want to `continue` if we don't have `hotPlugVFIO` _nor_ `coldPlugVFIO`, or did you have something different in mind?